bobweb/bob/command_gpt.py

```diff
@@ -134,25 +120,25 @@ async def generate_and_format_result_text(update: Update) -> string:
     if system_message_obj is not None:
         message_history.insert(0, system_message_obj)
 
-    # Uses OpenAi Http api as Openai python library version is too old and its upgrade is left for later development.
```

Comment:
So we won't move to python library? It could be up to date these days.

Reply:
That could be an option as well, but I find it easier to just use http API as there is no need to update dependencies when new API features are added. Of course using the library removes some boilerplate. Before this, the image creation was the only implementation using the library and others were already on the http API so I thought I should convert that to http API as well.